client/blocks/inline-help/inline-help-search-results.jsx

```diff
@@ -172,9 +158,11 @@ function HelpSearchResults( {
 		);
 	};
 
-	const renderSearchResultsSection = ( id, title, results ) => {
+	const renderSearchResultsSection = ( { type, title, results, condition } ) => {
 		/* eslint-disable jsx-a11y/no-noninteractive-element-to-interactive-role */
```

Comment:
This `eslint-disable` can be removed. The offending markup has probably been removed at some time.

Reply:
addressed in https://github.com/Automattic/wp-calypso/pull/55753/commits/a5c0545a5f0624f70ca0e132d64a0f27b61a9fdb